src/user.rs

```diff
@@ -29,18 +29,49 @@ impl Default for DeviceCreateOpts {
     }
 }
 
+/// Options that can be specified at when calling [`user_create`](trait.UserOps.html#tymethod.user_create).
+pub struct UserCreateOpts {
+    // see docs on `new`
+    needs_rotation: bool,
+}
+
+impl UserCreateOpts {
+    /// Constructor. Also see [`default`](#method.default).
+    ///
+    /// # Arguments
+    /// - `needs_rotation` - Set to true if the private key for this user should be rotated when the user "takes control" of their keys.
+    ///
+    /// The main use case for this is a workflow that requires that users and groups to be generated
+    /// prior to the user logging in for the first time. In this situation, a user's cryptographic identity
+    /// can be generated by a third party, like a server process, and then the user can take control of their
+    /// keys by rotating the private key using [`rotate_private_key`](TODO LINK).
```

Comment:
Maybe we should make an issue for this and link to that here.

Reply:
added a link to https://github.com/IronCoreLabs/ironoxide/issues/44